### PR TITLE
Add Vcs URI parsing

### PR DESCRIPTION
**Pull request by lumjjb**
Add Vcs URI parsing

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]

File: pkg/assembler/helpers/vcs.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
+//
```
Comment (mlieberman85) [inline pkg/assembler/helpers/vcs.go:1]: Should this live in a common helpers directory? Looking at this, this function is more or less the same one we use in the github collector to parse a URI that comes in as well.

Comment (lumjjb) [reply pkg/assembler/helpers/vcs.go:1]: yea, maybe a helper function to call out, and then leave the conversion to the graphql model in assembler helpers.

Comment (lumjjb) [reply pkg/assembler/helpers/vcs.go:1]: i think this could be put into a good first issue and handled as another PR?

Comment (lumjjb) [reply pkg/assembler/helpers/vcs.go:1]: https://github.com/guacsec/guac/issues/572

**Review by pxp928 — APPROVED** [review]
LGTM

**Comment by arorasoham9** [comment]
Is it alright if the functions if vcs.go and vcs_test.go are moved to misc?

**Comment by pxp928** [comment]
> Is it alright if vcs.go and vcs_test.go are moved to misc?

We can either move to misc or internal helper directory.

**Comment by arorasoham9** [comment]
Both the files lie in "pkg/assembler/helpers/" dir. Should I move them to misc or are they fine where they are?

**Pull request merged**